accelerator/core/response/ta_fetch_buffered_request_status.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2020 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+
+#ifndef RESPONSE_TA_fetch_buffered_request_status_H_
+#define RESPONSE_TA_fetch_buffered_request_status_H_
+
+#include "common/macros.h"
+#include "common/model/bundle.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @file accelerator/core/response/ta_fetch_buffered_request_status.h
+ */
+
+/**
+ * The enumeration of the status of buffered requests
+ */
+typedef enum {
+  UNSENT /**< Buffered request hasn't be broadcasted */,
+  NOT_EXIST /**< A request with given UUID is not existed */,
+  SENT /**< Buffered request has be broadcasted */
+} request_with_uuid_status_t;
```

Comment:
Should you add prefix for these enumerations?

Reply:
How about add prefix `MAM_`?

Reply:
Prefix `MAM` sounds fine.

Reply:
?